src/utils/big6Utils.ts

```diff
@@ -40,24 +40,28 @@ export const formatBig6USDPrice = (
   value: bigint = 0n,
   {
     compact = false,
+    compactDigits = false,
     fromUsdc = false,
     fullPrecision = false,
-  }: { compact?: boolean; fromUsdc?: boolean; fullPrecision?: boolean } = {},
+  }: { compact?: boolean; compactDigits?: boolean; fromUsdc?: boolean; fullPrecision?: boolean } = {},
```

Comment:
I like the second option, let's make this an optional digits arg and override the defaults if set. otherwise fallback to the existing `compact ? ...` logic

Reply:
I put it as an arg with a default and kept the compact logic to prevent side effects